### DISCUSSION TITLE
Group tables tabs

### DISCUSSION THREAD

**Discussion by Shlomixg** [dbeaver/dbeaver#36417] [Ideas]
Is it possible to add feature that will allow us to group tabs of elements like in Chrome?
Screenshot:
![image](https://github.com/user-attachments/assets/cd3e3932-e535-4ff0-9d63-d89d39b3ce40)

**Comment by Shlomixg** [comment]
Anyone?

**Comment by TB-HunBoka** [comment]
You can try this addon: https://marketplace.eclipse.org/content/vertical-tabs#details
It doesn't change the current tabs, but will give you a new window, where you can drag and drop your current tabs and create groups with it. These groups will be visible in this window only.
Dbeaver and Eclipse doesn't support this functionality by default.

**Comment by noa-hoogi** [comment]
I tried this but its not good as native support can be.
Any feedback from DBEAVER team ?🙏

**Reply by E1izabeth** [reply]
Similar feature requests #10235 #21485 
You can vote for them with 👍🏻  under the issue description to promote it to a higher priority.
